tap_mssql/sync_strategies/common.py

```diff
@@ -2,11 +2,19 @@
 # pylint: disable=too-many-arguments,duplicate-code,too-many-locals
 
 import copy
+import csv
```

Comment:
I think a few changes available in master aren't included here. The change to `logical.py` that updates the state even if no records changed is not here. See line 301 of `logical.py` in master - this isn't anywhere in this PR.